src/main/java/xyz/aadev/generitech/common/items/tools/ItemToolWrench.java

```diff
@@ -64,7 +64,7 @@
     private static IBlockState blockHarvest = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "blockHarvest" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1068)